### DISCUSSION TITLE
Score unavailable on TechEmpower

### DISCUSSION THREAD

**Discussion by elsandosgrande** [actix/actix-web#2905] [General] [answered]
Visiting the benchmark link in the README gives me this:
![image](https://user-images.githubusercontent.com/40576902/194768881-dfb33198-7b8b-4ff6-9184-8bd8d7606351.png)

Did something change since that link was added to the README?

**Comment by robjtede** [comment, marked as answer]
> Did something change since that link was added to the README?

Clearly so. I wasn't aware they'd removed our scores for round 20.

Left a comment here: https://github.com/TechEmpower/FrameworkBenchmarks/issues/4147#issuecomment-1272587707
